public/css/flags/flags.css

```diff
@@ -93,10 +93,14 @@
     background-position: -80px -11px;
 }
 
-.flag.flag-multiple {
+.flag.flag-id {
     background-position: -80px -22px;
 }
 
+.flag.flag-multiple {
+    background-position: -80px -33px;
```

Comment:
should be precentage rather than a static number ideally ( for different resolutions) but ill let it slide